moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -884,6 +884,13 @@ void ConfigurationFilesWidget::changeCheckedState(QListWidgetItem* item)
 
   // Enable/disable file
   gen_files_[index].generate_ = generate;
+
+  // Update GAZEBO_URDF_LOAD_ATTRIBUTE to load URDF file from config folder or to regenerate by xacro
+  if (gen_files_[index].write_on_changes == MoveItConfigData::SIMULATION)
```

Comment:
`write_on_changes` is a bitmask. Hence change to this:
```suggestion
  if (gen_files_[index].write_on_changes & MoveItConfigData::SIMULATION)
```

Reply:
Thank you! I was not sure about the best way, to find the related file.